bin/rbenv-ctags

```diff
@@ -71,6 +80,10 @@ generate_ctags_for() {
   fi
 }
 
+if [ -n "$ripper" ] && ! rbenv-which ripper-tags 2>&1 >/dev/null; then
+  exit "1"
```

Comment:
Presumably we want an error message like `echo "rbenv-ctags: ripper not found" >&2` before this.

`exit 1` would be the conventional way to write this (no quotes).

Reply:
Actually now that I think about this, we can't check for this globally; we need to check for it on a per-version basis. Which it looks like you've already accounted for.

Reply:
Yes. I will fix the quoting issue. If ripper-tags isn't installed, it just prints the

```
rbenv: ruby: command not found

The `ruby' command exists in these Ruby versions:
<...>
```
error, which seemed like it probably was the information we would want to convey?

Reply:
Yeah but you're exiting before calling it so that doesn't matter. I'm referring to this, a few lines up in a function:

```bash
if [[ "$languages" == "Ruby" ]] && [ -n "$ripper" ] && rbenv-which ripper-tags 2>&1 >/dev/null; then
```

You're checking for `ripper-tags`. The first time you silently exit. The second time you fall back to ctags. This sequence of events makes no sense.

Reply:
Good point. Will fix.

Reply:
I think I left the one in the function there knowing you would never get to it if the first check failed and we exited. I suppose I can just take it out then.

Reply:
> Actually now that I think about it, we can't check for `ripper-ctags` globally; we need to check for it on a per-version basis.

This is still true. Remove the global check, not the check inside the function.

Reply:
Right on. Should we exit in that case or fall back to `ctags`, maybe with a message saying we're doing so? If we don't fall back or exit, you'll get a `command not found` message for every file.

Reply:
Just checking in if this is looking alright now?

Reply:
No, the global check I told you to remove is still there:

```bash
if [ -n "$ripper" ] && ! rbenv-which ripper-tags 2>&1 >/dev/null; then
  exit 1
fi
```

What this does is check if **the currently activated Ruby version** has ripper-tags installed. If run `rbenv shell 2.7.0`, then `rbenv ctags --ripper 2.7.1 2.7.2`, it will check for the presence of `ripper-ctags` in 2.7.0, when what we care about is its presence in 2.7.1 and 2.7.2. You're checking a red herring, and then silently exiting based on the result.

Delete those 3 lines of code.

Reply:
> No, the global check I told you to remove is still there:
> 
> ```shell
> if [ -n "$ripper" ] && ! rbenv-which ripper-tags 2>&1 >/dev/null; then
>   exit 1
> fi
> ```
> 
> What this does is check if **the currently activated Ruby version** has ripper-tags installed. If run `rbenv shell 2.7.0`, then `rbenv ctags --ripper 2.7.1 2.7.2`, it will check for the presence of `ripper-ctags` in 2.7.0, when what we care about is its presence in 2.7.1 and 2.7.2. You're checking a red herring, and then silently exiting based on the result.
> 
> Delete those 3 lines of code.

I am once again asking you to remove this global check.